src/app/member/[memberIdentifier]/study/[studyIdentifier]/review/study-results.tsx

```diff
@@ -36,7 +63,7 @@ export const StudyResults: FC<{ latestJob: StudyJob }> = ({ latestJob }) => {
                 <Title order={4}>Study Results</Title>
```

Comment:
If there is no fingerprint associated with the user then there is no way they can decrypt the results. Maybe there's an easy way to tell them?